docs/conf.py

```diff
@@ -315,6 +315,7 @@
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     'python': ('https://docs.python.org/', None),
+    'marshmallow': ('http://marshmallow.readthedocs.io/en/latest/', None)
```

Comment:
Needed cuz otherwise it fails to build docs since `IndentifierSet` inherits from `List` which has a ref to `Fields`.